.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# default owners for everything in this repo
+*       @ajhfok @kellyma2 @kthommandra @aristanetworks/cloudvision-sre
```

Comment:
@devoxel this shows as "unknown owner" for me. I'm not sure if its because its incorrect, or I don't have the correct permissions to view the team https://github.com/orgs/aristanetworks/teams/cloudvision-sre. What does it show for you?

Reply:
Works now! @pidren changed the visibility of the team.